### PR TITLE
Ensure the Go examples in the cloud providers are valid go applications

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #4598

### CODE REVIEW

File: content/docs/intro/cloud-providers/aiven/_index.md [hunk 1]
```diff
@@ -76,13 +76,22 @@ import (
...
+    service, err := aiven.NewService(ctx, "test", &aiven.ServiceArgs{
+      Project:     pulumi.String("my-project"),
+      CloudName:   pulumi.String("google-europe-west1"),
+      Plan:        pulumi.String("startup-4"),
+      ServiceName: pulumi.String("my-service"),
+      ServiceType: pulumi.String("grafana"),
+    })
+    if err != nil {
+      return err
+    }
```
Comment (lukehoban) [inline content/docs/intro/cloud-providers/aiven/_index.md:90]: Indentation seems off?  Maybe tabs/spaces issue?

File: content/docs/intro/cloud-providers/akamai/_index.md [hunk 2]
```diff
@@ -84,16 +84,32 @@ import (
   "github.com/pulumi/pulumi-akamai/sdk/go/akamai"
   "github.com/pulumi/pulumi-akamai/sdk/go/akam/properties
```
Comment (lukehoban) [inline content/docs/intro/cloud-providers/akamai/_index.md:86]: Is there a missing `)` here?

File: content/docs/intro/cloud-providers/aiven/_index.md (hunk 1 above)
Comment (lukehoban) [reply content/docs/intro/cloud-providers/aiven/_index.md:90]: Quite a few different variants of these formatting issues throughout the rest of the PR it seems?

**Review by lukehoban — APPROVED** [review]
LGTM other than formatting issues.

**Comment by stack72** [comment]
@lukehoban I went through each example and ensured they now have been gofmt'ed

**Pull request merged**